rct229/rulesets/ashrae9012019/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/aggregate_min_OA_schedule_across_zones.py

```diff
@@ -0,0 +1,26 @@
+from operator import add
+
+
+def aggregate_min_OA_schedule_across_zones(zone_OA_CFM_list_of_schedules):
+    """Each HVAC system can serve multiple zones which can have multiple terminal units serving each zone. In order to obtain an hourly HVAC level OA CFM schedule the OA CFM provided by each zone needs to be aggregated for each hour of the year. This function receives an RMR (B, U, or P) and a list of zone OA min cfm schedules (generated by the get_min_OA_CFM_sch_zone function) and loops through each zone to create an aggregated 8760 for OA CFM for the HVAC system.
+
+    Parameters
+    ----------
+    zone_OA_CFM_list_of_schedules: list of list
+        A list which contains each zone's 8760 OA CFM schedule as a list value.
+
+    Returns
+    -------
+    list
+        This function receives an RMR (B, U, or P) and a list of zone OA min cfm schedules (generated by the get_min_OA_CFM_sch_zone function) and loops through each zone to create an aggregated 8760 for OA CFM for the HVAC system.
+    """
+
+    for zone_OA_CFM_list_of_schedule in zone_OA_CFM_list_of_schedules:
+        try:
+            hourly_zone_running_OA_CFM_total = list(
+                map(add, hourly_zone_running_OA_CFM_total, zone_OA_CFM_list_of_schedule)
```

Comment:
I am not sure if we want to have `add` from operator.
My main concern is from the `init` in the rulesets folder which may need to exclude these modules when filtering rule definition classes from the module.

Reply:
Great point! I just changed the code to use for loops to handle this.